### PR TITLE
add consumer for deleting user

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/831

### CODE REVIEW

File: src/ProjectService.Data/UserRepository.cs [hunk 1]
```diff
@@ -95,5 +95,16 @@ public List<DbProjectUser> Find(List<Guid> userIds)
         {
             return _provider.ProjectsUsers.Where(u => userIds.Contains(u.UserId)).ToList();
         }
+
+        public void Remove(Guid userId)
```
Comment (djino97) [inline src/ProjectService.Data/UserRepository.cs:99]: Why consumer named diactivate but repository method named as remove?

File: src/ProjectService.Broker/DisactivateUserConsumer.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            IUserRepository repository)
+        {
+            _repository = repository;
+        }
+
+        public Task Consume(ConsumeContext<IDisactivateUserRequest> context)
+        {
+            _repository.Remove(context.Message.UserId);
+
+            return Task.FromResult(0);
```
Comment (djino97) [inline src/ProjectService.Broker/DisactivateUserConsumer.cs:22]: Need return result executed operation

Comment (BelehovEgor) [reply src/ProjectService.Broker/DisactivateUserConsumer.cs:22]: We don't have to return anything here, becouse it is not for request client, it is for publish

File: src/ProjectService.Data/UserRepository.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/ProjectService.Data/UserRepository.cs:99]: in other repositories methods like this named "remove"

**Review by MariyaSin — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/ProjectService.Broker/DisactivateUserConsumer.cs (hunk 2 above)
Comment (djino97) [reply src/ProjectService.Broker/DisactivateUserConsumer.cs:22]: if use simple return

**Review by djino97 — APPROVED** [review]

**Pull request merged**